### DISCUSSION TITLE
AtlassianPlugin - JIRA

### DISCUSSION THREAD

**Discussion by nsd0hniehaus** [javamelody/javamelody#995] [Q&A]
Hi there,
we are using JavaMelody Plugin for monitoring our JIRA instance. To get information about database in Melody you have to configure database connection via JNDI in servcer.xml (https://github.com/javamelody/javamelody/wiki/AtlassianPlugin).
The configuration works great - but seems to be deprecated since JIRA Version 5: https://community.atlassian.com/t5/Jira-Software-questions/Database-monitoring-Connection-Pool-graph-no-values/qaq-p/1630762?utm_source=atlcomm&utm_medium=email&utm_campaign=solution_to_question&utm_content=topic
Is there any workaround? Maybe you should mention it in your docs?
Cheers
Holger

**Comment by evernat** [comment]
Hi Holger,

There is no workaround in order to have the monitoring of jdbc connections and of SQL requests in javamelody.
